source/uwp/Renderer/lib/AdaptiveInputChoiceSet.cpp

```diff
@@ -77,9 +77,24 @@ namespace AdaptiveCards { namespace XamlCardRenderer
     }
 
     _Use_decl_annotations_
-    HRESULT AdaptiveInputChoiceSet::get_Choices(IVector<IAdaptiveInputChoice*>** InputChoices)
+    HRESULT AdaptiveInputChoiceSet::get_Choices(IVector<IAdaptiveInputChoice*>** inputChoices)
     {
-        return m_InputChoices.CopyTo(InputChoices);
+        return m_InputChoices.CopyTo(inputChoices);
+    }
+
+    _Use_decl_annotations_
+    HRESULT AdaptiveInputChoiceSet::get_Id(HSTRING* speak)
```

Comment:
And in the other input classes

---
In reply to: [118370064](https://github.com/Microsoft/AdaptiveCards/pull/369#discussion_r118370064) [](ancestors = 118370064)